t/t5601-clone.sh

```diff
@@ -68,6 +68,14 @@ test_expect_success 'clone respects GIT_WORK_TREE' '
 
 '
 
+test_expect_success MINGW,CASE_INSENSITIVE_FS 'core.worktree is not added due to path case' '
+
+	mkdir CAPITALCASE &&
```

Comment:
Isn't it "UPPERCASE"?

Reply:
You're right. Fixed. I was thinking "capital letter case".